docs/sdk/src/external_resources.rs

```diff
@@ -12,3 +12,9 @@
 //! - [Polkadot Developers](https://github.com/polkadot-developers/)
 //! - [Polkadot Blockchain Academy](https://github.com/Polkadot-Blockchain-Academy)
 //! - [Polkadot Wiki: Build](https://wiki.polkadot.network/docs/build-guide)
+
+// Link References
```

Comment:
will the docs compile with this? I think these needs to be in the last line prefixed with `//!`.

Reply:
I fixed it in the script, 

can you please check above my query on .prdoc issue